CODEOWNERS

```diff
@@ -2,25 +2,27 @@
 *                                           @box/webapp
 
 # Root
-/*                                          @box/foundation @box/webapp
+/*                                          @box/foundation
 /LICENSE                                    @box/ui-elements
-/scripts/                                   @box/foundation @box/webapp
 /THIRD_PARTY_LICENSES                       @box/ui-elements
 
+# Build
+/scripts/                                   @box/foundation
+
 # Source
-/src/                                       @box/foundation @box/webapp
-/src/*                                      @box/create-communicate @box/preview @box/ui-elements
-/src/api/                                   @box/create-communicate @box/preview @box/ui-elements
-/src/api/Feed.js                            @box/collab @box/preview
+/src/                                       @box/webapp
+/src/*                                      @box/preview @box/ui-elements
+/src/api/                                   @box/preview @box/ui-elements
 /src/api/box-edit/                          @box/partners
-/src/api/tasks                              @box/collab @box/preview
-/src/elements/                              @box/create-communicate @box/preview @box/ui-elements
+/src/api/Feed.js                            @box/preview
+/src/api/tasks/                             @box/preview
+/src/api/uploads/                           @box/create-communicate
+/src/elements/                              @box/preview @box/ui-elements
 /src/elements/content-explorer/             @box/ui-elements
 /src/elements/content-open-with/            @box/partners
 /src/elements/content-picker/               @box/ui-elements
 /src/elements/content-preview/              @box/preview
 /src/elements/content-sidebar/              @box/preview
-/src/elements/content-sidebar/activity-feed @box/collab @box/preview
```

Comment:
There are some task directories in here that really should be reviewed by the collab team (things referenced in other task features outside the repo)